Sources/AuthFoundation/OAuth2/Configuration.swift

```diff
@@ -50,50 +51,20 @@ extension OAuth2Client {
         public init(issuerURL: URL,
                     discoveryURL: URL? = nil,
                     clientId: String,
-                    scope: String,
+                    scope: ClaimCollection<[String]>,
```

Comment:
Curious, why just not [String]?

Reply:
The `ClaimCollection` property wrapper abstracts interactions between values, types, conversions to/from raw values, etc.  Since this conforms to several `ExpressibleBy*Literal` protocols, this supports the assignment from a variety of convenience types a developer may supply.

The other convenience initializers are only necessary to account for `String` or `[String]` variables, which can't be assigned as literals.